_DeploymentAndDistroPackaging/ansible/cleanup/ciao.yml

```diff
@@ -17,16 +17,11 @@
   connection: local
   tasks:
     - name: Remove local files
-      local_action: file path={{ item }} state=absent
-      with_items:
-        - ./openrc
-        - ./admin-ciaorc
-        - ./demo-ciaorc
-        - ./clouds.yaml
-        - ./certificates
-        - ./images/.cnci_ready
-        - ./fetch
-        - /tmp/go
+      file: path={{ item }} state=absent
+      with_fileglob:
+        - /tmp/go/*
```

Comment:
Deploy host:  has no /tmp/go

Me:  mkdir /tmp/go
        put bad stuff in there

You: run ansible scripts
        use my bad stuff in /tmp/go

Is that a possible outcome?